dockers/docker-sonic-restapi/cert_monitor.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+'''
+This script monitors the certificate needed by rest-api container and safely restarts
+the process when it detects a certificate roll-over
+'''
+import os, subprocess, time
+from swsscommon import swsscommon
+from sonic_daemon_base.daemon_base import Logger
+import json
+
+RESTAPI_SERVER_PROCESS = "go-server-server"
+PATH_TO_CERT_TRACKER = "/restapi/cert_tracker"
+POLLING_FREQUENCY = 3600
+REDIS_TIMEOUT_MS = 0
+logger = Logger()
+
+def get_pid(process_name):
```

Comment:
check here. https://exceptionshub.com/how-to-get-pid-by-process-name-in-python.html

seems use pidof and check_output is simpler?